_layouts/agenda.html

```diff
@@ -4,12 +4,20 @@
 <div class="post">
   <header class="post-header">
     <h1 class="post-title">Agenda for {{ page.date | date: "%B %-d, %Y" }}</h1>
+    <h6 class="masthead">BAKERSFIELD COLLEGE</h6>
+    <h6 class="masthead">Computer Science Club</h6>
```

Comment:
The top of _every page_ already says **Bakersfield College Computer Science Club**. Is including it a second time redundant?